### PR TITLE
Refactor eth-offchain

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Should I change the `EthNetwork = EthRopsten`?

**Comment by HackFisher** [comment]
File conflicts need resolve first.

**Comment by AurevoirXavier** [comment]
Doesn't work after add mmr into header, no logs. I'll work on this tomorrow.

File: frame/chainrelay/eth/offchain/Cargo.toml [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
+simple-json = { version = '0.1.5', default_features = false, git = 'https://github.com/jimmychu0807/simple-json' }
 
+# github.com
 frame-support = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
 frame-system = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
 
-pallet-timestamp = { default-features = false,  git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
-pallet-eth-relay = { default-features = false, package = "darwinia-eth-relay", path = "../relay" }
+sp-runtime = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
+sp-std = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", tag = "v2.0.0-alpha.darwinia.1" }
```
Comment (HackFisher) [inline frame/chainrelay/eth/offchain/Cargo.toml]: Need to upgrade dependency to v2.0.0-alpha.darwinia.2

Comment (AurevoirXavier) [reply frame/chainrelay/eth/offchain/Cargo.toml]: already on my local

**Pull request merged**